DBM-Core/modules/Sounds.lua

```diff
@@ -105,6 +110,7 @@ do
 
 	local hasCached = false
 	local cachedTable
+	---@deprecated Use new utility functions
```

Comment:
Should we remove the other deprecated notice on the line below then?

Reply:
done